assets/style/main.css

```diff
@@ -0,0 +1,517 @@
+  background-position: 0 0;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-bottom-left-radius: 15%;
+  margin-bottom: 40px;
+}
+
+.intro-container {
+  margin: 0 20px;
+  width: 70%;
+}
+
+.link-img {
+  height: 20px;
+  width: 20px;
+  margin: 10px;
+}
+
+.list-social {
+  display: flex;
+  align-items: flex-end;
+}
+
+.work-container {
+  display: flex;
+  align-items: stretch;
+  border-radius: 20px;
+  margin: 60px;
+}
+
+.mlr-20 {
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+.mall-30 {
+  margin: 30px;
+}
+
+.section-img {
+  width: 50%;
+  border-radius: 8px;
+}
+
+.section-content {
+  width: 50%;
+  border-radius: 8px;
+}
+
+.img-content {
+  width: 100%;
+  height: 100%;
+}
+
+.flexr-default {
+  display: flex;
+  justify-items: center;
+}
+
+.mt-30 {
+  margin-top: 30px;
+}
+
+.work-link {
+  margin: 1px;
+  padding: 10px;
+}
+
+.flex-list {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.dot-icon {
+  width: 10px;
+  height: 10px;
+  background-color: #c1c7d0;
+  border-radius: 50%;
+}
+
+.lang-link {
+  margin: 2px 5px;
+  padding: 8px;
+  background-color: #ebebff;
+  border-radius: 8px;
+}
+
+.btn-more {
+  margin: 10px;
+  padding: 12px;
+  background-color: #fff;
+  border: 1px solid #396df2;
+  border-radius: 12px;
+}
+
+.btn-more:hover {
+  color: #fff;
+  border: 1px solid #7f8cff;
+  background-color: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  border-radius: 8px;
+}
+
+.btn-more:disabled {
+  background-color: #fff;
+  border: 1px solid #5e6c84;
+  color: #5e6c84;
+}
+
+.btn-more:active {
+  background: #2230d2;
+  border-radius: 8px;
+  color: #fff;
+}
+
+.grid-item-3 {
+  border-top-right-radius: 50px;
+  margin: 30px;
+}
+
+.about-container {
+  display: flex;
+  align-items: flex-start;
+  margin: 30px;
+}
+
+.about-myself {
+  width: 45%;
+  margin: 30px;
+}
+
+.about-myexperience {
+  width: 55%;
+  margin: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.experience-list {
+  width: 90%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 20px;
+  margin-bottom: 10px;
+}
+
+.upward-icon {
+  rotate: 90deg;
+}
+
+.exper-lang {
+  display: flex;
+}
+
+.exper-lang-list {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.lang-img {
+  margin-bottom: 4px;
+  margin-right: 8px;
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+}
+
+.list-hozin {
+  width: 92%;
+  border-bottom: 0.5px solid #dfe1e6;
+}
+
+.exper-link {
+  padding: 5px 10px;
+  background-color: #f7f7f9;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  border-radius: 8px;
+}
+
+.grid-item-4 {
+  background-image: url("../images/desktop/contact.png");
+  background-position: 0 0;
+  background-repeat: no-repeat;
+  background-size: cover;
+  border-bottom: 0;
+}
```

Comment:
- [ ] According to the given template, there should be some shapes in the `background image` of the `contact me` section. In your design, there are no shapes. Kindly fix the design.

***Given Design***
![image](https://user-images.githubusercontent.com/60083437/221146586-87d8bf38-464c-4116-bdff-01f054281196.png)

***Your Design***
![image](https://user-images.githubusercontent.com/60083437/221146669-19913fd2-284e-4bda-ac0b-596fddd64651.png)